node/utils/simulation.ts

```diff
@@ -39,14 +65,24 @@ export const orderFormItemToSeller = (
   orderFormItem: OrderFormItem & {
     paymentData: any
     ratesAndBenefitsData: RatesAndBenefitsData
+    logisticsInfo: any[]
   }
 ) => {
+  const unitMultiplier = orderFormItem.unitMultiplier ?? 1
+  const [logisticsInfo] = orderFormItem.logisticsInfo
+
   const commertialOffer = {
-    Price: orderFormItem.sellingPrice / 100,
+    Price: orderFormItem.sellingPrice
+      ? Number((orderFormItem.sellingPrice / (unitMultiplier * 100)).toFixed(2))
```

Comment:
`sellingPrice` isn't accurate inside the orderFormItem, can we use `priceDefinition.calculatedSellingPrice` instead?